ansible/roles/homeassistant/defaults/main.yml

```diff
@@ -162,6 +162,13 @@ homeassistant__basic_auth_name: 'homeassistant'
 #
 # List of user accounts which will be allowed access to Home Assistant.
 homeassistant__basic_auth_users: '{{ ansible_local.core.admin_users|d([]) }}'
+
+                                                                   # ]]]
+# .. envvar:: homeassistant__webserver_user [[[
+#
+# Name of the webserver user account which will be granted read only access to
+# the www directory outside of the Home Assistant configuration directory.
+homeassistant__webserver_user: '{{ ansible_local.nginx.user|d("www-data") }}'
```

Comment:
```suggestion
homeassistant__webserver_user: '{{ ansible_local.nginx.user | d("www-data") }}'
```

Reply:
Thanks for reviewing. I would say the omitted spaces are backed by [DEP 2 - DebOps code standards](https://docs.debops.org/en/master/dep/dep-0002.html).